packages/smart-contracts/test/contracts/EthConversionProxy.test.ts

```diff
@@ -0,0 +1,212 @@
+import { ethers, network } from 'hardhat';
+import {
+  EthereumFeeProxy__factory,
+  EthConversionProxy__factory,
+  EthereumFeeProxy,
+  ChainlinkConversionPath,
+  EthConversionProxy,
+} from '../../src/types';
+import { BigNumber, Signer } from 'ethers';
+import { expect, use } from 'chai';
+import { solidity } from 'ethereum-waffle';
+import { CurrencyManager } from '@requestnetwork/currency';
+import { chainlinkConversionPath } from '../../src/lib';
+
+use(solidity);
+
+describe('contract: EthConversionProxy', () => {
+  let from: string;
+  let to: string;
+  let feeAddress: string;
+  let signer: Signer;
+  const smallAmountInFIAT = BigNumber.from('100000000'); 
+  const smallerAmountInFIAT = BigNumber.from('10000000');
```

Comment:
why not `amountInFiat` and `feesAmountInFiat`? any reason for the `small` qualification? does it have something to do with the test setup?